osm2streets/src/output.rs

```diff
@@ -145,75 +170,56 @@ impl StreetNetwork {
             }
 
             // Draw arrows along oneway roads.
-            if road.oneway_for_driving().is_some() {
-                for (lane, center) in road.lane_specs_ltr.iter().zip(lane_centers.iter()) {
-                    if !lane.lt.is_for_moving_vehicles() {
-                        continue;
-                    }
-
-                    let step_size = Distance::meters(20.0);
-                    let buffer_ends = Distance::meters(5.0);
-                    let arrow_len = Distance::meters(1.75);
-                    let thickness = Distance::meters(0.16);
-                    for (pt, angle) in center.step_along(step_size, buffer_ends) {
-                        let arrow = PolyLine::must_new(vec![
-                            pt.project_away(arrow_len / 2.0, angle.opposite()),
-                            pt.project_away(arrow_len / 2.0, angle),
-                        ])
-                        .make_arrow(thickness * 2.0, geom::ArrowCap::Triangle);
-                        output.push(PaintArea::new(arrow.into()));
-                    }
-                }
-            }
-
-            // Add stripes to show buffers. Ignore the type of the buffer for now -- we need to
-            // decide all the types and how to render them.
             for (lane, center) in road.lane_specs_ltr.iter().zip(lane_centers.iter()) {
-                if !matches!(lane.lt, LaneType::Buffer(_)) {
+                if !lane.lt.is_for_moving_vehicles() {
                     continue;
                 }
 
-                // Mark the sides of the lane clearly
-                let thickness = Distance::meters(0.16);
-                output.push(PaintArea::new(
-                    center
-                        .must_shift_right((lane.width - thickness) / 2.0)
-                        .make_polygons(thickness)
-                        .into(),
-                ));
-                output.push(PaintArea::new(
-                    center
-                        .must_shift_left((lane.width - thickness) / 2.0)
-                        .make_polygons(thickness)
-                        .into(),
-                ));
-
-                // Diagonal stripes along the lane
-                let step_size = Distance::meters(3.0);
+                // Add arrows along the lane, starting at the end.
+                let step_size = Distance::meters(20.0);
```

Comment:
Hmm, we still have to make locale / rendering type decisions here? It'd maybe be neat to say something like "this lane has this symbol spaced along it somehow" and have the rendering layer decide specifics, especially if that layer is the one interpreting locale more. But this is OK too.

Reply:
Hmm, good point.